src/transformers/modeling_lxmert.py

```diff
@@ -46,6 +49,96 @@ def __init__(self):
     def forward(self, x):
         return gelu(x)
 
+@dataclass
+class LxmertModelOutput(ModelOutput):
```

Comment:
Moved the model outputs in this file, as they're model-specific. Renamed and re-ordered the parameters.